front/app/components/PostShowComponents/Comments/Comment/CommentReplyButton.tsx

```diff
@@ -59,9 +57,8 @@ interface Props {
   postType: 'idea' | 'initiative';
   commentId: string;
   commentType: 'parent' | 'child' | undefined;
-  authUser: GetAuthUserChildProps;
   author?: IUserData;
-  post: IIdeaData | IInitiativeData;
+  idea?: IIdeaData;
```

Comment:
We never used `post` as it being a proposal, only an idea, so it didn't make sense to pass `IInitiativeData`.

Reply:
🤔 I think this is used when replying to a proposal too. If that is the case, then we will need `IInitiativeData` and the implementation to handle a reply on a comment on a proposal. At the moment, that doesn't work

Reply:
I left the check for `idea` in the same place as I did when it was still `post`, which is why it was not working for proposals.

Thanks for pointing out! But `IInitiativeData` is not used in this component. Only `IIdeaData`. :-)